server/database/database.js

```diff
@@ -29,20 +30,26 @@ const ReviewSchema = mongoose.Schema({
 });
 const Review = mongoose.model('Review', ReviewSchema);
 
+
 const MovieSchema = mongoose.Schema({
   id: Number,
+  //need to modify later like reviews
+  suggested: Array,
+  trending: Array,
+  history: Array,
```

Comment:
Is this how to write it in Mongoose? I am not sure if I was correct on line 18 putting it like [String] since it is supposed to be an array.

Reply:
putting it in [String] - i believe you are assigning a data type of an array of strings.  So yes, if you want something to hold an array of strings [string] would be the proper way to go about it